roles/container_runtime/tasks/docker_storage_setup_overlay.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Setup the docker-storage for overlay
+  template:
+    src: overlay2_dss.j2
+    dest: /etc/sysconfig/docker-storage-setup
+    owner: root
+    group: root
+    mode: 0664
+  when:
+  - container_runtime_run_docker_storage_setup | default(False)
+  - container_runtime_docker_storage_driver == 'overlay2'
```

Comment:
I think we should change this from _driver to _type.  This will allow us greater flexibility to control more types of storage scenarios.  For instance, if a storage driver can be used in a variety of ways (overlay2 on top of gluster) or what have you.

Reply:
Sounds good